os2forms_payment.services.yml

```diff
@@ -1,13 +1,16 @@
 services:
+  logger.channel.os2forms_payment:
+    parent: logger.channel_base
+    arguments: [ 'os2forms_payment' ]
+
   Drupal\os2forms_payment\Helper\PaymentHelper:
+    class: Drupal\os2forms_payment\Helper\PaymentHelper
```

Comment:
This line (`class: …`) is superfluous when you use the class name as service id:

```suggestion
```